data/issues/emissions/emission_data_calculations.py

```diff
@@ -205,52 +204,62 @@ def calculate_hit_net_zero(df):
 
 
 def calculate_budget_runs_out(df):
-    last_year = CURRENT_YEAR  # Year of the last datapoint
+    # Year from which the budget applies (after correction with respect to reported years since budget start)
+    last_year = max(LAST_YEAR_WITH_SMHI_DATA, BUDGET_YEAR)  
```

Comment:
last_year används omväxlande för LAST_YEAR_WITH_SMHI_DATA och max(LAST_YEAR_WITH_SMHI_DATA, BUDGET_YEAR), kanske ha två olika namn så att koden blir enklare att hänga med i? tänker även att first_year skulle kunna vävas in i det så det blir konsekvent

Reply:
Se commit: https://github.com/Klimatbyran/klimatkollen/commit/b4b52bdd1bbda902675baae2c043972a37203d5b

Reply:
Känns det mer begripligt?

Reply:
Ja underbart!